.tekton/operator-bundle-pipeline.yaml

```diff
@@ -105,64 +105,114 @@ spec:
     description: This sets the expiration time for intermediate OCI artifacts produced and used during builds after which they can be garbage collected.
     name: oci-artifact-expires-after
     type: string
+
   - name: operator-image-repo
     description: Image repository for the StackRox Operator.
     type: string
+    default: "quay.io/rhacs-eng/stackrox-operator"
+  - name: operator-image-downstream-repo
+    description: Downstream repository for the StackRox Operator.
```

Comment:
While it's not incorrect to say "downstream" in this context, subjectively, more than half of engineers would like to never hear about "downstream" and not know anything about it.
As we onboard Konflux builds, there will be no "downstream" in its original sense and so we have good reasons to not keep dragging that terminology into the future.

These are user-facing repositories in the Red Hat Container Catalog where we ultimately release containers. I suggest to choose a name based on that. (I have a suggestion which I'm not very fond of and would prefer you to suggest before I offer mine.)

Reply:
Please just suggest the name you have in mind.

Reply:
`operator-image-catalog-repo`
`operator-image-rh-catalog-repo`
`operator-image-prod-catalog-repo`

Reply:
Aligned on 'catalog-repo'.

Reply:
Params updated in f4b7d81f40163f40e80c383ba1ec69a50715c515 and 14773a1b1d3ff2207e09f85e35d5728230f0260f.